app/src/ui/app-error.tsx

```diff
@@ -91,6 +94,51 @@ export class AppError extends React.Component<IAppErrorProps, IAppErrorState> {
     }, dialogTransitionLeaveTimeout)
   }
 
+  private renderErrorWithMetaDataFooter(error: ErrorWithMetadata) {
+    const { retryAction } = error.metadata
+    if (retryAction !== undefined) {
+      if (retryAction.type === RetryActionType.Clone) {
+        return this.renderRetryCloneFooter(retryAction)
+      }
+    }
+
+    if (isGitError(error.underlyingError)) {
+      return this.renderGitErrorFooter(error.underlyingError)
+    }
+
+    return this.renderDefaultFooter()
+  }
+
+  private renderRetryCloneFooter(retryAction: RetryAction) {
```

Comment:
This function is not using the `retryAction` param so we can get rid of it

Reply:
Good catch, this method was initially called `renderRetryFooter` and I had intended it to be the entry point for all retry actions but then I renamed it at the last minute and didn't update the assumptions within.